src/common/signing.props

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildType Condition="Exists('..\..\script\src\MetricsService.cs')">Internal</BuildType>
```

Comment:
lol yeah that would probably be smarter, I just randomly picked one of the files in there that weren't likely to move :P